src/adapters/mediawiki.spec.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+import {persist} from './mediawiki';
+
+QUnit.module('Adapter');
+
+QUnit.test('MediaWiki persist()', function(assert) {
+	var data = 'foo';
+	var promise = persist(data);
```

Comment:
this will need to be updated to include `persistOptions` after https://github.com/Wikia/infobox-template-builder/pull/15 is merged.